### ISSUE TITLE
Highlight parent comment when typing a response

### ISSUE THREAD

**Issue by gvx** [euphoria-io/heim#77]
I keep making the mistake of replying to the wrong parent when in a conversation. Would it make sense to visually highlight the parent comment? I realise the line to the parent comment is already highlighted, but it's just not noticeable enough for me.

**Comment by NotAMoose** [comment]
I agree that it could be bit more clear. We're working on a front end [rewrite](https://alpha.euphoria.io/), which I think should probably help this issue. I don't think it will be clear how much it solves the issue until the alpha site styling is rolled out live and folks have started using it.
